### PR TITLE
Remove checksum-maven-plugin && Remove -Padvanced

### PR DESCRIPTION
**Pull request by cliu123**
*Issue #, if available:*
#739 
*Description of changes:*
1. Remove checksum-maven-plugin from pom.xml
2. Remove -Padvanced from ci.yml, cd.yml and ReadMe.md.
3. Remove the advanced profile and move it’s executions into the regular build package goal for the maven assembly plugin.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by vrozov** [comment]
Please remove `advanced` profile from pom.xml. Note that it is necessary to generate plugin .zip file.

File: pom.xml [hunk 1]
```diff
@@ -829,114 +829,52 @@
     </build>
     <profiles>
         <profile>
-            <id>advanced</id>
```
Comment (sujithvm) [inline pom.xml:832]: We cannot just delete the advanced profile. We need to move the executions of maven assembly plugin

File: pom.xml [hunk 2]
```diff
@@ -829,114 +829,52 @@
...
+            <id>veracode</id>
```
Comment (sujithvm) [inline pom.xml:832]: Out of scope for this PR but `veracode` profile can be removed too

File: pom.xml (hunk 1 above)
Comment (vrozov) [reply pom.xml:832]: There should be also a way to suppress creation of .zip file (just generate .jar).

**Pull request closed without merging**